package.json

```diff
@@ -9,14 +9,14 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
+    "test": "ember try:each"
```

Comment:
for two reasons:

* what you said
* and it's what new ember addon projects generated by the cli tool also do